sdk/pull-requests.md

```diff
@@ -14,6 +14,34 @@ This document is intended as a canonical reference documenting how the SDK Team
 - Ideally, in most cases, PRs should be landed (merged) to the `main` branch by the Lead Engineer responsible for that client library
 - Avoid publishing comments into that public domain that just represent housekeeping or reminders to other team members - these should more appropriately be handled as internal messaging over Slack (probably via the appropriate SDK repository channel)
 
+## Etiquette
+
+### Conversations
+
+- Conversations should be closed by the person who started them
+- If you do need to close a conversation on behalf of someone else then you should tag them in a new comment in that conversation thread stating that you are closing the conversation on their behalf, stating your reasons for doing that - valid reasons include:
+  - that person is known not to be available at the moment
+  - there is urgency and conversations need to be resolved so that the PR can be merged (required by branch protect rule for target branch)
+- For authors adopting [The Merge Approach](#flexible-regarding-merge-from-or-rebase-atop-target) to PR lifecycle event creation, it is really helpful to reviewers to keep your subsequent commits to the PR's underlying branch small and focussed so that you can directly link to the commit that resolves any given conversation in a comment in that conversation thread. Reviewers are often coming back into a PR completely out of context, so anything the author can do to help them understand the specific thing they need to look at to understand how the PR has evolved should be encouraged.
```

Comment:
Maybe replace one of those 'understand's in the final, lengthy sentence with 'comprehend'?

Reply:
Actually I'd add: please refrain from changing from Draft to Ready For Review if your CI status is not green...

Reply:
https://github.com/ably/engineering/pull/16/commits/4d45739f9b676e0ea1207373f000e922b022b38b